source/_integrations/weatherflow.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: "WeatherFlow"
+description: "WeatherFlow weather sensor"
```

Comment:
This description differents from most others, please take a look at other integration pages how those are generally looking.